test/units/testsuite-58.sh

```diff
@@ -868,6 +868,8 @@ testcase_exclude_files() {
 Type=root-${architecture}
 CopyFiles=/
 CopyFiles=/zzz:/
+CopyFiles=/:/oiu
+ExcludeFiles=+/oiu/usr
```

Comment:
I know this is a bit of a bikeshedding exercise, but shouldn't this be a separate option? This overloads the ExcludeFile pattern to not just change how the option works for a path, but actually makes it work on a different path. Wouldn't it be clearer to have ExcludeTargetFiles= option instead?

Reply:
I followed systemd-nspawn's --bind option here, which does the same thing more or less with the '+' modifier

Reply:
Updated to do this via a new option ExcludeFilesTarget= instead.